### PR TITLE
v1 Prototype

### PR DESCRIPTION
**Pull request by mastastealth**
No pictures yet, but please make sure I don't sound/read like a Colombian.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/wombatsecurity/wombatsecurity.github.io/pull/45" target="_blank">ReviewNinja

File: _posts/2015-12-16-proto-eryting.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: Prototype All The Things
+categories: development
+tags: development
+author_github: mastastealth
+date:   2015-12-16 09:00:00
+comments: true
+banner_image: proto.png
+---
+
+Throughout the years of coding, you begin picking up habits and adjusting workflows for maximum efficiency. I can recall one of the earliest things for me in this regard was the addition of [Emmet](http://emmet.io/) (previously Zen Coding) into my editors and after getting accustomed to it seeing the huge boosts in speed while generating HTML and CSS. These little things in your workflow are generally based off personal experience, and everyone has their own methods of doing things. Shortly after the [adquisition](https://www.wombatsecurity.com/threatsim-acquisition) I had to elaborate one of my methods with my new team members, which I've decided to go ahead and share here as well.
```
Comment (laurawhalin) [inline _posts/2015-12-16-proto-eryting.md]: you have acquisition misspelled in brackets for the link

File: _posts/2015-12-16-proto-eryting.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<!--more-->
+
+## The Process
+It's time to build out a new component, a new page, or a whole new app for your sit, what do you do? First, you need to get your requirements down. What the new "thing" needs to do, what's it for, etc. When you have some basic parameters down, generally the first thing that comes out of that are [wireframes](https://en.wikipedia.org/wiki/Website_wireframe). Wireframes help create that initial skeleton of "where things go" on the page. At Wombat, wireframes are where the iterations go down; various versions are mocked up and tweaks are made along with other departments' input till a more finalized version of the wireframe is decided upon. With the final nod of approval and any tweaks that to need to be snuck in, a ticket is created to begin the coding process.
```
Comment (laurawhalin) [inline _posts/2015-12-16-proto-eryting.md]: I think you meant "whole new app for your site"?

File: _posts/2015-12-16-proto-eryting.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## The Difference
+As the "new guy" to the Wombat Development family I unsurprisingly had a difference in my process which initially seemed to have caused some confusion, and understandably so. The important steps along the process to point out above are that it went "requirements -> **wireframes** -> revisions -> code" and what I was/am accustom to at this point was "requirements -> **wireframe** -> code -> revisions". As you can imagine, when someone starts throwing code out earlier than expected it can seem like you're jumping a huge part of the established process. Amusingly, the difference wasn't as jarring as it initially seemed.
```
Comment (laurawhalin) [inline _posts/2015-12-16-proto-eryting.md]: small change: "I was/am accustomed to"

File: _posts/2015-12-16-proto-eryting.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+* **Versions** - It's slightly more difficult to code different "versions" of the same page in a working prototype since generally a "link" goes to **one** place.
+* **Speed** - Getting to a speed that makes prototyping worth it over comps might take a while. You definitely shouldn't expect a junior developer to code up a mockup page as fast as a more experienced designer whipping up wireframes.
+* **Impression** - While the team gets accustom to it, it can be difficult to initially remember that the prototype is *meant* to be just as "incomplete" as a wireframe. The illusion that things are clickable, coded, and working can sometimes give the impression "this is how it will be" and that is certainly not the case. A prototype should be able to be completely nuked or redesigned if necessary.
```
Comment (laurawhalin) [inline _posts/2015-12-16-proto-eryting.md]: change to accustomed

File: _posts/2015-12-16-proto-eryting.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+
+Now of course, if prototyping early were so inefficient developers wouldn't bother with it, but there are definitely pros which I personally believe outweigh any of the (generally overcomeable) issues:
+
+* **Speed** - This time, in the good way. When you can code up prototypes at a decent speed it helps to immediately understand some of the workload coming you way. How long certain areas take, what areas might take longer, and depending on what you're prototyping and how, you might be able to use a good portion of your code and refine it for production!
```
Comment (laurawhalin) [inline _posts/2015-12-16-proto-eryting.md]: change to "coming your way"

**Comment by laurawhalin** [comment]
Even when I read it in my head with a Colombian accent, it didn't come off as too Colombian. I think you're good! Nice perspective! :pencil2:

**Comment by mastastealth** [comment]
@laurawhalin Updated!

**Pull request merged**